Assets/Scripts/VrSdk.cs

```diff
@@ -184,19 +184,35 @@ void Awake()
             if (App.Config.m_SdkMode == SdkMode.Oculus)
             {
 #if OCULUS_SUPPORTED
-      // ---------------------------------------------------------------------------------------- //
-      // OculusVR
-      // ---------------------------------------------------------------------------------------- //
-      OVRManager manager = gameObject.AddComponent<OVRManager>();
-      manager.trackingOriginType = OVRManager.TrackingOrigin.FloorLevel;
-      manager.useRecommendedMSAALevel = false;
-
-      SetControllerStyle(TiltBrush.ControllerStyle.OculusTouch);
-      // adding components to the VR Camera needed for fading view and getting controller poses.
-      m_VrCamera.gameObject.AddComponent<OculusCameraFade>();
-      m_VrCamera.gameObject.AddComponent<OculusPreCullHook>();
-
-      gameObject.AddComponent<OculusMRCCameraUpdate>();
+                // ---------------------------------------------------------------------------------------- //
+                // OculusVR
+                // ---------------------------------------------------------------------------------------- //
+                OVRManager manager = gameObject.AddComponent<OVRManager>();
+                manager.trackingOriginType = OVRManager.TrackingOrigin.FloorLevel;
+                manager.useRecommendedMSAALevel = false;
+
+                SetControllerStyle(TiltBrush.ControllerStyle.OculusTouch);
+
+                // Set custom refresh rate, if possible. To see the current framerate, look in the logs for prints like this one. The first number is the actual frame rate, the second is the display refresh rate (aka target)
+                // VrApi   : FPS=90/90,Prd=33ms,Tear=0,Early=0...
+                float targetDisplayRefresh = App.UserConfig.Video.DisplayRefresh;
+                float[] freqs = OVRManager.display.displayFrequenciesAvailable;
+                foreach (float freq in freqs)
```

Comment:
I wasn't sure if we should do this check here or in UserOptions, but it needs the `OVRManager` and that was throwing exceptions in UserOptions, so I left it here